src/test/java/org/apache/commons/lang3/builder/ToStringStyleTest.java

```diff
@@ -16,10 +16,10 @@
  */
 package org.apache.commons.lang3.builder;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Hi.
I don't think the changes in ToStringStyleTest.java is necessary...

Reply:
@XenoAmess fixed, thank you!